kite-tools-parent/kite-tools/src/test/java/org/kitesdk/cli/commands/TestCSVImportCommand.java

```diff
@@ -196,15 +196,10 @@ public void testIncompatibleSchemaFieldType() throws Exception {
 
     // This will fail because NaN isn't a valid long and the field is required
     command.targets = Lists.newArrayList("target/incompatible.csv", datasetName);
-    TestHelpers.assertThrows("Should complain about schema compatibility",
```

Comment:
I'm fine merging this with this test simply ignored. We should add a follow-up to show the error message correctly.